lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -244,9 +216,9 @@ namespace Js
     }
 #endif // DBG
 
-    void JsBuiltInEngineInterfaceExtensionObject::EnsureJsBuiltInByteCode(ScriptContext * scriptContext)
+    void JsBuiltInEngineInterfaceExtensionObject::EnsureSourceInfo()
```

Comment:
The SourceInfo is created the first time bytecode is loaded and re-used if the other 2 files are loaded later.